lib/matplotlib/style/core.py

```diff
@@ -84,6 +85,73 @@ def use(style):
                 raise IOError(msg % style)
 
 
+def get_substyles(styles):
```

Comment:
Sorry, but I don't really understand what's going on here. Is this adding a new `"style"` parameter to stylesheets that allows you to specify a child stylesheet? You can't get a tree from that since it's just a single value, right? Also, I would think it would make more sense for a `"style"` parameter to point to a parent stylesheet.